Game/Entities/Boss.cs

```diff
@@ -0,0 +1,65 @@
+using Godot;
+using System;
+
+public partial class Boss : RigidBody2D, IDamageable
+{
+	[Export]
+	private Player player;
+	[Export]
+	private World world;
+
+	private Timer FireTimer;
+	private Timer AltFireTimer;
+	public override void _Ready()
+	{
+		FireTimer = GetNode<Timer>("FireTimer");
```

Comment:
Instead of needing to initialize these like this, use autostart and wait time in the editor properties (means you don't need a reference to the timer in code at all)